tests/csm/rest/test_capacity_quota.py

```diff
@@ -0,0 +1,442 @@
+        for iam_user in self.created_iam_users:
+            resp = s3_misc.delete_iam_user(iam_user[0], iam_user[1], iam_user[2])
+            if resp:
+                iam_deleted.append(iam_user)
+            else:
+                self.log.error("IAM deletion failed for %s ", iam_user)
+        self.log.info("IAMs deleted %s", iam_deleted)
+        for iam in iam_deleted:
+            self.created_iam_users.remove(iam)
+        assert_utils.assert_true(len(self.buckets_created) == 0, "Bucket deletion failed")
+        assert_utils.assert_true(len(self.created_iam_users) == 0, "IAM deletion failed")
+        self.log.info("[ENDED] ######### Teardown #########")
+
+    @pytest.mark.skip("Feature not ready")
+    @pytest.mark.lc
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-40632')
+    def test_40632(self):
+        """
+        Test set & get API for User level quota & capacity admin user - enable
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("Step 1: Creating IAM user with login as manage user")
+        payload = self.csm_obj.iam_user_payload_rgw("random")
+        resp = self.csm_obj.create_iam_user_rgw(payload)
+        self.log.info("Verify Response : %s", resp)
+        assert_utils.assert_true(resp.status_code == HTTPStatus.CREATED,
+                                 "IAM user creation failed")
+        self.created_iam_users.add(resp.json()['tenant'] + "$" + payload["uid"])
+        resp1 = self.csm_obj.compare_iam_payload_response(resp, payload)
+        self.log.info("Printing response %s", resp1)
+        assert_utils.assert_true(resp1[0], resp1[1])
+        self.log.info("Step 2: Create bucket under above IAM user")
+        self.akey = resp.json()["access_key"]
+        self.skey = resp.json()["secret_key"]
+        self.bucket_name = "iam-user-bucket-" + str(int(time.time()))
+        self.log.info("Verify Create bucket: %s with access key: %s and secret key: %s",
+        self.bucket_name, self.akey, self.skey)
+        bucket_created = s3_misc.create_bucket(self.bucket_name, self.akey, self.skey)
+        assert bucket_created, "Failed to create bucket"
+        self.log.info("Step 3: Perform PUT API to set user level quota with random values")
+        uid = resp1.json()['tenant'] + "$" + payload["uid"]
+        test_cfg = self.csm_conf["test_40632"]
+        max_size = test_cfg["max_size"]
+        max_objects = test_cfg["max_objects"]
+        resp3 = self.quota_obj.set_user_quota(uid, "user","true", max_size, max_objects)
+        assert_utils.assert_true(resp3[0],resp3[1])
+        self.log.info("Step 4: Perform GET API to get user level quota")
+        resp4 = self.quota_obj.get_user_quota(uid, "user")
+        assert_utils.assert_true(resp4[0], resp4[1])
```

Comment:
Where is the verification for the get quota response matches SET quota?
Can we move SET and GET quota with verification in new function call?
Function which verifies that whatever is SET is visible in GET along with status code

Reply:
created the functions